MOM6-interface/CMakeLists.txt

```diff
@@ -71,12 +77,15 @@ if(MOM6SOLO)
   add_executable(mom6solo ${mom6_solo_src_files})
   add_dependencies(mom6solo mom6_obj)
   set_target_properties(mom6solo PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mod_solo)
-  target_include_directories(mom6solo PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>)
-  target_include_directories(mom6solo PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod_solo>)
+  target_include_directories(mom6solo PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>
+                                              $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod_solo>
+                                              $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/../stochastic_physics>)
```

Comment:
If `mom6solo` now depends on `stochastic_physics` target (line 87), is it necessary to explicitly add `../stochastic_physics` as an include directory. It shouldn't